.github/workflows/yaml-checks.yaml

```
@@ -0,0 +1,29 @@
+name: YAML checks
+
+on:
+  push:
+    branches:
+      - 'main'
+      - '[0-9]+.[1-9][0-9]*.x'
+    paths:
+      - '**.yaml'
+      - '**.yml'
+  pull_request:
+    branches:
+      - 'main'
+      - '[0-9]+.[1-9][0-9]*.x'
+    paths:
+      - '**.yaml'
+      - '**.yml'  
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install yamllint
+        run: pip install yamllint
+
+      - name: Lint YAML files
+        run: yamllint .github docs
```

please us the make target here
```suggestion
      - name: Lint YAML files
        run: make yamllint
```

---

Done!